kubetest/util.go

```diff
@@ -140,12 +141,20 @@ func finishRunning(cmd *exec.Cmd) error {
 
 	finished := make(chan error)
 
+	sigChannel := make(chan os.Signal, 1)
+	signal.Notify(sigChannel, os.Interrupt)
+
 	go func() {
 		finished <- cmd.Wait()
 	}()
 
 	for {
 		select {
+		case <-sigChannel:
+			log.Printf("Killing %v(%v) after receiving signal", stepName, -cmd.Process.Pid)
```

Comment:
after receiving interrupt?

Reply:
Yeah, "interrupt" is correcter than "signal" strictly, but I did choose "signal" because we could cover another signals in the future when we need and "signal" is not so bad on that.
This is not a strong opinion at all.